addons/mrp/report/mrp_report_mo_overview.py

```diff
@@ -696,7 +696,7 @@ def _get_reserved_qty(self, move_raw, warehouse, replenish_data):
         if not replenish_data['qty_reserved'].get(move_raw):
             total_reserved = 0
             wh_location_ids = self._get_warehouse_locations(warehouse, replenish_data)
-            linked_moves = self.env['stock.move'].browse(move_raw._rollup_move_origs()).filtered(lambda m: m.location_id.id in wh_location_ids or m.location_dest_id.id not in wh_location_ids)
+            linked_moves = self.env['stock.move'].browse(move_raw._rollup_move_origs()).filtered(lambda m: m.location_id.id in wh_location_ids)
```

Comment:
can't we use `not m.is_subcontract`?

Reply:
One issue I see with this is that `is_subcontract` is defined in the `mrp_subcontracting`, requiring a new override in this module. While I agree this modification is about the subcontracting, it seems a bit overkill to me.
Also, the problematic move is the one from Production -> Subcontracting Location, which is not `is_subcontract` (that would be its dest move)